src/ui/public/vis/map/service_settings.js

```diff
@@ -65,9 +71,13 @@ uiModules.get('kibana')
 
         this._loadFileLayers = _.once(async () => {
           const catalogue = await this._loadCatalogue();
-          const fileService = catalogue.services.filter((service) => service.type === 'file')[0];
+          const fileServices = catalogue.services.filter((service) => service.type === 'file');
```

Comment:
I think this could be shortened, by using the `.find` method?

```js
const fileService = catalogue.services.find(service => service.type === 'file');
if (!fileService) {
  return [];
}
```